src/fxos/read_gaia_commit.js

```diff
@@ -0,0 +1,23 @@
+let adb = require('../adb');
+let debug = console.log.bind(console, '[fxos/read_gaia_commit]');
+let denodeify = require('denodeify');
+let exec = require('mz/child_process').exec;
+let readFile = require('mz/fs').readFile;
+let tmpdir = denodeify(require('tmp').dir);
+
+module.exports = async function readGaiaCommit() {
+  let local = await tmpdir();
+  debug('tmpdir', local);
+  await adb.pull(
+    '/system/b2g/webapps/settings.gaiamobile.org/application.zip',
+    local
+  );
+
+  debug('unzip application.zip');
+  await exec('unzip application.zip', {cwd: local});
```

Comment:
I know a lot of the code in here is currently platform-specific, but it would be nice if we could use a cross-platform module to handle the unzipping in the future.